wp-content/themes/citylimits/partials/nav-sticky.php

```diff
@@ -91,80 +51,85 @@ class="input-medium appendedInputButton search-query" value="" name="s" />
 										<span class="icon-bar"></span>
 										<span class="icon-bar"></span>
 									</div>
+									<span>Menu</span>
 								</a>
 							</li>
 						</ul>
-
-					</div>
-
-					<!-- BEGIN MOBILE MENU (hidden on desktop) -->
-					<div class="nav-left">
-						<ul>
-							<li class="home-icon"><a href="<?php echo esc_url( home_url( '/' ) ); ?>"></a></li>
-						</ul>
 					</div>
-					<!-- END MOBILE MENU -->
 
 					<!-- BEGIN DESKTOP MENU -->
 					<div class="nav-shelf">
-					<ul class="nav">
-						<?php if ( of_get_option('sticky_header_logo') !== '') { ?>
-							<li class="home-icon">
-								<a href="<?php echo esc_url( home_url( '/' ) ); ?>">
-									<?php
-									if ( of_get_option( 'sticky_header_logo' ) !== '' )
-										largo_home_icon( 'icon-white' , 'orig' );
-									?>
-								</a>
-							</li>
-						<?php } else { ?>
-							<li class="site-name"><a href="/"><?php echo $site_name; ?></a></li>
-						<?php }
-
-							/* Build Main Navigation using Boostrap_Walker_Nav_Menu() */
-							$args = array(
-								'theme_location' => 'main-nav',
-								'depth'		 => 0,
-								'container'	 => false,
-								'items_wrap' => '%3$s',
-								'menu_class' => 'nav',
-								'walker'	 => new Bootstrap_Walker_Nav_Menu()
-							);
-							largo_nav_menu($args);
+						<div class="close-menu toggle-nav-bar">
+							<span class="dashicons dashicons-no-alt"></span>
```

Comment:
We should probably put an `aria-label="close menu"` here. https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_aria-label_attribute

Reply:
Yep. Added in https://github.com/INN/umbrella-citylimits/pull/65/commits/b0f2bbe93ff23440ab0059fa308e32cc06df4942